src/librustc_driver/lib.rs

```diff
@@ -371,8 +369,6 @@ pub fn run_compiler(
                     });
 
                     result
-                    // AST will be dropped *after* the `after_analysis` callback
-                    // (needed by the RLS)
```

Comment:
If you remove that comment, can you move the drop of the AST where earlier?

Reply:
Will do!

Reply:
Done